x-pack/plugins/ml/public/components/full_time_range_selector/index.test.tsx

```diff
@@ -30,7 +28,7 @@ describe('FullTimeRangeSelector', () => {
   };
 
   const query: Query = {
-    language: QueryLanguageType.KUERY,
+    language: 'kuery',
```

Comment:
Hey Stacey, can you explain why this is no longer available as an `enum`?

Reply:
ideally it would be, it's just that the type in the "source of truth" package always used a string, while the embeddable's one used an enum.  A follow up PR could be be converting the source of truth type to use an enum.  I thought about adding it in this PR... but these PRs really quickly grow in scope so tried to stick with one goal: stop importing/exporting these from the embeddables plugin.

Reply:
No comments other than this one :)